### ISSUE TITLE
Make Conventional Commits the default preset

### ISSUE THREAD

**Issue by ondrejsevcik** [semantic-release/semantic-release#3406]
### New feature motivation

Hello,

First of all, I would like to thank you for such an amazing tool.

I was curious whether it would make sense to make the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) the default preset.

The Angular preset does not allow for marking breaking changes in a commit message title. It's a major drawback. Also I think the community is more aware of the conventional commit style rather than Angular commit style.

There has been some discussion in [this PR](https://github.com/semantic-release/commit-analyzer/issues/231#issuecomment-2127394159) to make the Conventional Commits the default, so I thought it would be good to bring it up in a separate issue.

Thank you for considering this.

### New feature description

Change the default preset to conventionalcommits.

**Comment by ondrejsevcik** [comment]
@travi is there any way i could help with that? Are there any specific steps that need to happen before this change is made?

**Comment by AndrewTriesToCode** [comment]
@travi  Is this still a priority?

**Comment by babblebey** [comment]
This is very much still priority.... We are looking to make this the default preset

In the meantime, you can manually set it as preset in your config and installing the preset package

```
"plugins": [
    ["@semantic-release/commit-analyzer", { "preset": "conventionalcommits" }]
]
```

Install the preset [conventional-changelog-conventionalcommits](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-conventionalcommits)

**Comment by felipecrs** [comment]
I believe the only remaining bit is https://github.com/semantic-release/semantic-release/pull/1836.
